### DISCUSSION TITLE
Neo4j to pytorch-geometric

### DISCUSSION THREAD

**Discussion by archids** [pyg-team/pytorch_geometric#5184] [Q&A] [answered]
Hello All:

Can anyone recommend few tutorials / code examples to import graphs from Neo4j database to pytorch-geometric showing the complete process?

Thanks so much!

**Comment by rusty1s** [comment, marked as answer]
Hopefully, [this](https://towardsdatascience.com/integrate-neo4j-with-pytorch-geometric-to-create-recommendations-21b0b7bc9aa) tutorial is useful to you.

**Reply by archids** [reply]
Hello Matthias
Thanks so much. I will check it out.
